obj/mainDashBoard.js

```diff
@@ -0,0 +1,17 @@
+const functions = {
+    waitForPage: function () {
+      return this.waitForElementVisible('@providerName');
+    },
+    orderFirstitem: function () {
+        this.click('@firstMenuItem');
+        this.click('@confimOrderBtn');
+    },
+  }
+  module.exports = {
+    commands: [functions],
+    elements: {
+        providerName: '.headline.red--text',
```

Comment:
This is very much depending that provider is marked as red provider. ".headline" is unique and should be enough in this case